tests/mfx_dispatch/linux/CMakeLists.txt

```diff
@@ -37,12 +37,18 @@ add_executable(mfx_dispatch_test
   mfx_dispatch_test_mocks.cpp
   mfx_dispatch_test_fixtures.cpp)
 
+target_link_libraries( mfx_dispatch_test mfx gtest gmock pthread )
+
+target_include_directories( mfx_dispatch_test PUBLIC ../../../api/mfx_dispatch/linux )
+
 set_target_properties(mfx_dispatch_test PROPERTIES
   RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BIN_DIR}/${CMAKE_BUILD_TYPE})
 
-target_link_libraries( mfx_dispatch_test mfx gtest gmock pthread )
+add_test(NAME run_mfx_dispatch_test
```

Comment:
should this block be protected w/ if(RUN_TESTS)?

Reply:
This will only have effect if enable_testing() had been invoked, which is guarded w/ if (RUN_TESTS)

Reply:
ok.